pkg/httperr/default_error.go

```diff
@@ -0,0 +1,32 @@
+package httperr
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mittwald/api-client-go/mittwaldv2/generated/schemas/commonsv2"
+)
+
+func IsDefaultErrorResponse(res *http.Response) (*commonsv2.Error, bool) {
+	target := commonsv2.Error{}
+
+	if err := json.NewDecoder(res.Body).Decode(&target); err != nil {
+		return nil, false
+	}
```

Comment:
I think this might also succeed with an empty object (or one with other properties than `.type` and `.message`). Maybe add another check to assert that `target.Type` and `target.Message` are actually populated with values?

```suggestion
	}
	
	if target.Type == "" || target.Message == "" {
		return nil, false
	}
```